### ISSUE TITLE
track uploaded but not shown

### ISSUE THREAD

**Issue by c-e-p-x-u-o** [openstreetcam/openstreetview.org#176]
2018-08-14 I've uploaded the track

now it is not shown in the app nigher on the site.

**Comment by alexandrui-telenav** [comment]
Did it show up in the meantime?  If not, can you share your OSC username?

**Comment by alexandrui-telenav** [comment]
Realized you're &ergio, all your tracks should be visible, can you confirm?

**Comment by c-e-p-x-u-o** [comment]
Yes, I'm &ergio, and no, I don't see the track that I've uploaded on 2018-08-14.

**Comment by c-e-p-x-u-o** [comment]
Could you check logs?

**Comment by Skippern** [comment]
I have several tracks (17) which on the dashboard shows as "uploading", oldest is more than a year ago, and latest was this July. They have been uploaded with script and possibly not terminated correctly. Would help if it was possible to finalize upload from web dashboard or by some form of API command. Also, there should be a timeout on such uploads, so with no changes to the upload within a time, i.e. a week, the upload should be automatically finalized.
